libp2p/protocols/identify.nim

```diff
@@ -76,6 +77,14 @@ proc encodeMsg*(peerInfo: PeerInfo, observedAddr: Multiaddress): ProtoBuffer
   else:
     peerInfo.agentVersion
   result.write(6, agentVersion)
+
+  ## Optionally populate signedPeerRecord field.
+  ## See https://github.com/libp2p/go-libp2p/blob/master/p2p/protocol/identify/pb/identify.proto for reference.
```

Comment:
Could you put a permalink instead of master, always a pain when a link dies

Reply:
Yeah, thanks! Updated links in 0d5dfbb